mlflow/models/__init__.py

```diff
@@ -26,6 +26,49 @@
 from ..utils.environment import infer_pip_requirements
 from .evaluation import evaluate, EvaluationArtifact, EvaluationResult, list_evaluators
 
+from mlflow.models.flavor_backend_registry import get_flavor_backend
+from mlflow.utils import env_manager as _EnvManager
+
+
+def build_docker(
```

Comment:
Introduce an API equivalent of `mlflow models build-docker` so that users don't have to call `mlflow.sagemaker.build_and_push_container()` with special "no_push" args in order to build an MLflow Model Server docker image. This `build_and_push_container` should arguably be moved to `push_container` since it duplicates work done by `mlflow models build-docker`, but that can happen in a follow-up PR.

Reply:
This is tested sufficiently by `mlflow docker` CLI tests, since the CLI calls this method directly. See https://github.com/mlflow/mlflow/blob/8098785831a3414bed62ec3239b60a6a500f4dbd/tests/models/test_cli.py#L416-L492